ci/infrastructure/pipeline.yml

```
@@ -9,25 +9,39 @@ groups:
   - bosh-cleanup
   - fetch-latest-stemcell
   - deploy-prometheus
+  - deploy-multiapps-controller
 
 - name: others
   jobs:
   - set-pipeline
 
 resource_types:
-- name: gcs-resource
-  type: registry-image
+- name: file-url
+  type: docker-image
   source:
-    repository: frodenas/gcs-resource
+    repository: pivotalservices/concourse-curl-resource
+    tag: latest
 
 resources:
+- name: multiapps-controller-web-war
+  type: file-url
+  source:
+    url: https://repo.maven.apache.org/maven2/org/cloudfoundry/multiapps/multiapps-controller-web/1.174.0/multiapps-controller-web-1.174.0.war
+    filename: multiapps-controller-web-1.174.0.war
+
+- name: multiapps-controller-web-manifest
+  type: file-url
+  source:
+    url: https://repo.maven.apache.org/maven2/org/cloudfoundry/multiapps/multiapps-controller-web/1.174.0/multiapps-controller-web-1.174.0-manifest.yml
+    filename: multiapps-controller-web-1.174.0-manifest.yml
+
 - name: ci
   type: git
   icon: github
   source:
     uri: git@github.com:cloudfoundry/app-autoscaler-release
     private_key: ((autoscaler-deploy-key-private))
-    branch: main
+    branch: feature/739-setup-mtar-controller
```

let's not forget to revert that

```suggestion
    branch: main
```

---

done